homeassistant/components/prowl/strings.json

```diff
@@ -17,5 +17,15 @@
     "abort": {
       "already_configured": "API key is already configured"
     }
+  },
+  "issues": {
+    "prowl_yaml_deprecated": {
+      "title": "Prowl YAML configuration deprecated",
+      "description": "Configuring Prowl using YAML has been deprecated. An automatic import of your existing configuration has been made.\nPlease remove the Prowl configuration from configuration.yaml and update any automations to use the new `notify.send_message` action exposed with this new entity."
```

Comment:
> nd update any automations to use the new `notify.send_message` action exposed with this new entity.

Uh that sounds like a major breaking change or something that should be been raise repairs for separately; and not something that should be squeezed into this message.

Reply:
Yes, we do that too. https://github.com/home-assistant/core/pull/154727/files#diff-16a6406edebe12b764d2e6c7ccc098fdb11b4c7bcfcf1ab6edc46ce9f416f7ecR81 I just added it here to reinforce that there are more actions other than just migrating the YAML.
Should I perhaps phrase it better?